feat: public roadmap document (#466)
ory/docs: docs/docs/early-access.mdx

title: Ory Cloud Roadmap
---

This document outlines our product development plans for Ory Cloud for the next
quarters. Please note that our roadmap is subject to change therefore do not
treat the information below as fixed. Our Product Development Team is actively
working on the roadmap, triaging, and (re)prioritizing to deliver the best value
for our customers.

:::note

Ory Cloud is actively being developed and is currently in early access and
invite-only.

:::

## Roadmap

Users are often asking when a particular feature will be implemented or when it

> can be expected to be available. This document provides our high level plans in
> a more time frame focus manner.

Reviewer: ```suggestion
can be expected to be available. This document outlines high-level plans for the Ory Cloud.
```

We will also create Github Project Board in
[Ory Cloud GitHub repository](https://github.com/ory/cloud) with issues and

> timeliness reflecting our plan presented down below. Issues will provide more

Reviewer: ```suggestion
timelines reflecting our plan presented down below. Issues will provide more
```
